tests/Endpoint_selenium_unittest.py

```diff
@@ -0,0 +1,142 @@
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+import unittest
+import re
+import sys
+import os
+
+
+# first thing first. We have to create product, just to make sure there is atleast 1 product available
+# to assign endpoints to when creating or editing any.
+# importing Product_selenium_unittest as a module
+try:  # First Try for python 3
+    import importlib.util
+    product_unit_test_module = importlib.util.spec_from_file_location("product_selenium_unittest",
+        os.path.join(os.environ['DD_ROOT'], 'tests', 'product_selenium_unittest.py'))
+    product_unit_test = importlib.util.module_from_spec(product_unit_test_module)
+    product_unit_test_module.loader.exec_module(product_unit_test)
+except:  # This will work for python2 if above fails
+    import imp
+    product_unit_test = imp.load_source('product_selenium_unittest',
+        os.path.join(os.environ['DD_ROOT'], 'tests', 'product_selenium_unittest.py'))
```

Comment:
DD_ROOT is al alias for 'dojo'. However, 'tests' is not is dojo, so this will not work. It can be assumed that all these unit tests will remain together in the same file. With that, we can find the relative path the current file, and just append file to be referenced.

dir_path = os.path.dirname(os.path.realpath(__ file __))
    product_unit_test = imp.load_source('product_selenium_unittest',
        os.path.join(dir_path + '/product_selenium_unittest.py'))

Reply:
This has now been implemented and working fine